ui/app/AppLayouts/Profile/views/wallet/EditNetworkView.qml

```diff
@@ -13,9 +13,15 @@ ColumnLayout {
     property var networksModule
     property var combinedNetwork
     property var networkRPCChanged
+    property bool areTestNetworksEnabled: false
 
     signal evaluateRpcEndPoint(string url, bool isMainUrl)
-    signal updateNetworkValues(int chainId, string newMainRpcInput, string newFailoverRpcUrl, bool revertToDefault)
+    signal updateNetworkValues(int chainId, bool testNetwork, string newMainRpcInput, string newFailoverRpcUrl, bool revertToDefault)
+
+    onVisibleChanged: {
```

Comment:
You could use a `Binding` (in order not to break with manual tab changes) here but assuming the first tab will be always `prod` this is good imo :)